docs/source/deploying-hpc.rst

```diff
@@ -54,24 +58,41 @@ these systems like this.
    from dask.distributed import Client
    client = Client(cluster)    # Connect to that cluster
 
-Dask-jobqueue provides a lot of possibilities like adaptive dynamic scaling
+By submitting many small jobs into a cluster it is possible to get allocations
```

Comment:
Might be worth mentioning that some clusters actually prioritize large jobs (not common, but I've seen it). This makes using Jobqueue actually problematic. Sometimes the number of jobs allowed in the running state can be limited too.

Reply:
Good points, I've updated things with this info.